experimental/uefi/kernel/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_std]
+#![feature(lang_items)]
+#![feature(alloc_error_handler)]
+
+mod avx;
+pub mod i8042;
+mod libm;
+mod logging;
+mod memory;
+mod serial;
+
+use core::panic::PanicInfo;
+use log::{error, info};
+use rust_hypervisor_firmware_subset::{boot, paging, pvh};
+
+pub fn rust64_start(rdi: &pvh::StartInfo) -> ! {
```

Comment:
I think it would be nice to add some doc comments to the public functions here.

Reply:
Done.